addons/l10n_ec_website_sale/views/website_sales_templates.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="partner_info" name="Ecuadorian partner">
+
+        <!-- show identification type -->
+        <div t-attf-class="mb-3 #{error.get('l10n_latam_identification_type_id') and 'o_has_error' or ''} col-xl-6">
+            <label class="col-form-label" for="l10n_latam_identification_type_id">Identification Type</label>
+            <t t-if="partner.can_edit_vat()">
+                <select name="l10n_latam_identification_type_id" t-attf-class="form-select #{error.get('l10n_latam_identification_type_id') and 'is-invalid' or ''}">
+                    <option value="">Identification Type...</option>
+                    <t t-foreach="identification_types or []" t-as="id_type">
+                        <option t-att-value="id_type.id" t-att-selected="id_type.id == int(identification) if identification else id_type.id == partner.l10n_latam_identification_type_id.id">
+                            <t t-esc="id_type.name"/>
+                        </option>
+                    </t>
+                </select>
+            </t>
+            <t t-else="">
+                <p class="form-control" t-esc="partner.l10n_latam_identification_type_id.name" readonly="1" title="Changing Identification type is not allowed once document(s) have been issued for your account. Please contact us directly for this operation."/>
```

Comment:
This is acceptable for the MVP. In the long run however it is expected that the user can set a different VAT for each transaction, for instance he wants to use is account to buy for his grandma at one time, for him at a second time, for the company he works for at a third time. It is a feature available in lots of ecommerces and mobile apps in Ecuador.

I understand this is hard to achieve as the vat is a commercial field that is in sync with the parent partner.

Reply:
We can leave this for later, and we see if we receive feedbacks about it. We haven't had this on other localizations so far.